<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Swap3</class>
 <widget class="QDialog" name="Swap3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wanna swap?</string>
  </property>
  <widget class="QPushButton" name="pbtn_noswap">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Not swap</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_playmore">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play more</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_swap">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Swap</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>320</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You are on the defensive position.&lt;/p&gt;&lt;p&gt;Do you wanna choose to swap or not,&lt;/p&gt;&lt;p&gt;or play 2 more pieces?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
